Sources/ConsoleKit/Terminal/Terminal.swift

```diff
@@ -13,10 +13,13 @@ public final class Terminal: Console {
 
     /// Dynamically exclude ANSI commands when in Xcode since it doesn't support them.
     internal var enableCommands: Bool {
+        if let override = self.stylizedOutputOverride {
```

Comment:
should we rename this, since `override` is a swift keyword?

Reply:
Fixed 🙂